org.gnome.Epiphany.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id" : "org.gnome.Epiphany",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "epiphany",
+    "finish-args" : [
+        "--device=dri",
+        "--filesystem=/run/.heim_org.h5l.kcm-socket",
+        "--filesystem=xdg-download",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--system-talk-name=org.freedesktop.GeoClue2"
```

Comment:
Missing notification and secret service.

Reply:
Ah indeed, just noticed the commit upstream

Reply:
I think secrets depends on xdg-desktop-portal 1.6, I would keep that until 3.38

Reply:
The secrets situation will suck, since that will just be lost data.

Reply:
Notifications permission is needed for 3.36, not needed for 3.38.

Secrets permission is... controversial. I think it's probably better to not add it, and users will just  (hopefully?) receive errors if xdg-desktop-portal is not new enough. Because otherwise, when users upgrade xdg-desktop-portal to the new version with the secrets portal, all their passwords will disappear. And that's probably even worse than having the password manager broken.

Created https://github.com/flatpak/xdg-desktop-portal/issues/483 for that.

Maybe if a migrator is added in the future, then it would make more sense to support older portal versions, but in the meantime I think outright broken is probably better than data loss.

Reply:
> Notifications permission is needed for 3.36, not needed for 3.38.
> 
> Secrets permission is... controversial. I think it's probably better to not add it, and users will just (hopefully?) receive errors if xdg-desktop-portal is not new enough. Because otherwise, when users upgrade xdg-desktop-portal to the new version with the secrets portal, all their passwords will disappear. And that's probably even worse than having the password manager broken.
> 
> Created [flatpak/xdg-desktop-portal#483](https://github.com/flatpak/xdg-desktop-portal/issues/483) for that.
> 
> Maybe if a migrator is added in the future, then it would make more sense to support older portal versions, but in the meantime I think outright broken is probably better than data loss.

Makes sense.